documents/scope.md

```diff
@@ -0,0 +1,20 @@
+# Scope and planning
+
+## Stage 1
+
+Estimated completion 2024.
+
+Based on discussion from [2024-04-02](https://docs.google.com/document/d/1a-777RetR4k2gZKHJ5pG3apWKt86jV7-vU-QChX7lv4/edit?usp=sharing) our stage 1 plan is:
+
+1. Focus on a small set of nouns, sourced from CLDR, e.g. month names, units etc to facilitate faster prototyping.
+2. Gender and plural inflection can be handled by current message formatting logic, we can tackle them in later stages.
```

Comment:
Can you clarify what is stated here?

The plural rules are already handled by CLDR plural rules, but inflecting a word to the plural form seems to be a big reason for this group. That's not needed for month names, but inflecting a quantity requires the plural form. Inflecting to the plural form is required for handling units (mentioned above).  Handling units also requires handling grammatical case.

I'm fine with deferring the ability to inflect the word "actor" to "actress" and vis versa, but I'd like to at least allow annotation of the gender of a given word, which is required for articles and prepositions.

Reply:
Added a section to clarify. Properly casing a noun includes plural form generation, also at least in Serbian you need gender and additional info to be able to case the noun - so that info would be part of the lexicon.

Reply:
> That's not needed for month names...

Maybe not for month names, but for day names, I could see things like "Three Thursdays from now" being something somebody would want to produce...